crates/gen/src/types/delegate.rs

```diff
@@ -36,7 +36,9 @@ impl Delegate {
         let abi_signature = signature.gen_winrt_abi(gen);
         let fn_constraint = signature.gen_winrt_constraint(gen);
         let guid = self.0.gen_guid(gen);
-        let phantoms = self.0.gen_phantoms(gen);
+        let intf_struct_phantoms = self.0.gen_phantoms(gen);
```

Comment:
I'd prefer to call this simply `struct_phantoms` - that's sufficiently unambiguous.